### PR TITLE
Improvement for  input types of the makeVoiceCall function message

### PR DESCRIPTION
**Pull request by SanduDS**
Resolves wso2-enterprise/choreo#3584

## Approach
1) Added new record type for use input type
2) Updated the test cases
3) Updated the samples

## Automation tests
 - Unit tests 
   > Done
 - Integration tests
   > Done

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: twilio/types.bal [hunk 1]
```diff
@@ -272,3 +272,17 @@ public type MessageResourceResponse record {
...
     json subresourceUris = {};
 };
+
+# Represents voice call message input options
+#
+# + userInput - A Twiml URL or an inline Message that what should be heard when the other party picks up the phone 
+# + userInputType - Whether the userInput is a URL or inline message  
+public type VoiceCallInput record {
+     string userInput;
+     VoiceCallInputType userInputType;
```
Comment (abeykoon) [inline twilio/types.bal:282]: Shall we take Type first and then the input? Then at Choreo, rendered form will have type first as a combo box and then the input. WDYT?

File: twilio/endpoint.bal [hunk 2]
```diff
@@ -141,21 +141,27 @@ public client class Client {
...
+    # + voiceCallInput - What should be heard when the other party picks up the phone (a Url that returns TwiML Voice instructions or 
+    #            inline message. example: "http://demo.twilio.com/docs/voice.xml" or "Thank you for calling")
     # + statusCallback - (optional) StatusCallback record which contains the callback url and the events whose status 
     #                     needs to be delivered.
     # + return - If success, returns voice call response object with basic details, else returns error
     @display {label: "Make a voice call"}
     remote isolated function makeVoiceCall(@display {label: "Caller Number"} string fromNo, 
                                            @display {label: "Callee Number"}string toNo, 
-                                           @display {label: "TwiML URL"} string twiml, 
+                                           @display {label: "Voice Message and Type"} VoiceCallInput voiceCallInput,
```
Comment (abeykoon) [inline twilio/endpoint.bal:152]: won't `Input` be enough? We know operation is `Make a voice call` and the generated form will have `type` and `input` fields.

Comment (SanduDS) [reply twilio/endpoint.bal]: Yes, I did the requested change

File: twilio/types.bal (hunk 1 above)
Comment (SanduDS) [reply twilio/types.bal:281]: Yes, I changed the order.  :+1:

**Review by abeykoon — APPROVED** [review]

**Pull request merged**